core/src/main/java/com/gentics/mesh/core/data/root/impl/SchemaContainerRootImpl.java

```diff
@@ -141,6 +141,11 @@ public SchemaContainer create(InternalActionContext ac, SearchQueueBatch batch,
 		MeshAuthUser requestUser = ac.getUser();
 		SchemaModel requestModel = JsonUtil.readValue(ac.getBodyAsString(), SchemaModelImpl.class);
 		requestModel.validate();
+
```

Comment:
Don't throw an error. Instead use default `false` if the container flag is omitted.